app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewModel.kt

```diff
@@ -365,17 +367,17 @@ class AttendeeViewModel(
         )
     }
 
-    fun loadUser(id: Long) {
-        if (id == -1L) {
+    fun loadUser() {
+        if (getId() == -1L) {
```

Comment:
store the value of getId in a variable and use it. Rather than calling the function 3 times